src/MusicStore/Program.cs

```diff
@@ -66,6 +70,39 @@ public static void Main(string[] args)
                 requestTime.Stop();
                 Console.WriteLine("Response: {0}", response.StatusCode);
                 Console.WriteLine("Request took {0}ms", requestTime.ElapsedMilliseconds);
+                
+                var minRequestTime = long.MaxValue;
+                var maxRequestTime = long.MinValue;
+                var averageRequestTime = 0.0;
+
+                Console.WriteLine("Running 100 requests");
+                for(int i = 1; i <= 100; ++i)
+                {
+                    requestTime.Restart();
+                    response = client.GetAsync("http://localhost:5000").Result;
+                    requestTime.Stop();
+
+                    Console.WriteLine("Response: {0}", response.StatusCode);
+                    Console.WriteLine("Request took {0}ms", requestTime.ElapsedMilliseconds);
```

Comment:
Do we really need to print the timings of each request? That's going to be a monster load of output.